odoo_module_migrate/migration_scripts/migrate_170_180.py

```diff
@@ -0,0 +1,53 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
this file looks a copy past of this Odoo/odoo file https://github.com/odoo/odoo/blob/master/odoo/upgrade_code/17.5-00-tree-to-list.py

- copyright and author should be conserved.
- you have not right to change the licence of the code. (the current code is LGPL, not AGPL)

what do you think ?

Reply:
About the licence, I'm not sure, but I think that should be ok if we remove that line and mention the author in a comment with a link to the original code.

Reply:
Well. Regarding the licence of this part of code, there is no choice. This code is LGPL. Except if odoo allow you to change licence, you have to mention that licence in that file.

Reply:
@legalsylvain I’ve updated the file as per your suggestions:

Restored the license to LGPL-3.0.
Added a comment attributing the original method replace_tree_with_list_in_views from the source: [Odoo 17.5-00-tree-to-list.py](https://github.com/odoo/odoo/blob/master/odoo/upgrade_code/17.5-00-tree-to-list.py).

Please let me know if this looks good to you.